doc/man/task-sync.5.in

```diff
@@ -5,123 +5,91 @@ task-sync \- A discussion and tutorial for the various task(1) data
 
-.SH OPTION 1: VERSION CONTROL SYSTEMS
-There are several good, distributed VCS systems (git, hg, ...) and centralized
-VCS systems (svn, cvs ...), and they all function in a similar fashion for our
-purposes.
+.SH TERMINOLOGY
 
-Setup is straightforward.  You place your .task directory under revision
-control.  You then need to perform a regular commit/push/pull/update to make
-sure that the data is propagated when needed.  You can even do this using shell
-scripts so that every task command is preceded by a 'pull' and followed by
-a 'push'.
+.SH LOCAL SYNCHRONIZATION
 
-Strengths:
-.br
-  - Good data transport mechanisms
-.br
-  - Secure transport options
-
-Weaknesses:
-.br
-  - You need proficiency with VCS tools
-.br
-  - You will need to manually resolve conflicts frequently
-.br
-  - You need to provide the mechanism for making sure copies are up to date
+Taskwarrior can sync your tasks locally. While this setup does not make your
+tasks accessible from multiple systems, it does provide a convenient backup and
+allow internal storage to be reused. If you later decide to sync to a server,
+the transition is as simple as changing all replicas to use the server
+synchronization settings explained in the next section.
 
+Set 'sync.local.server_dir' to the directory where the synchronized data should
+be stored:
 
-.SH OPTION 2: FILE SHARING SERVICES
-There are many file sharing services, such as DropBox, Amazon S3, Google Drive,
-SkyDrive and more.  This technique involves storing your .task directory in a
-shared directory under the control of the file hosting services.
+    $ task config sync.local.server_dir /path/to/sync
 
-Syncing happens quickly, although it is possible to run into conflict situations
-when there is no network connectivity, and the tasks are modified in two
-separate locations.  This is because the file hosting service knows only about
-files, and it has no idea how to merge tasks.  Avoid this problem by never
-modifying the same task on two machines, without an intervening sync.
+Then run 'task sync' to synchronize to this local "server".
 
-Setup simply involves creating the directory and modifying your data.location
-configuration variable like this:
+.SH SERVER SYNCHRONIZATION
 
-    $ task config data.location /path/to/shared/directory
+Configure each replica which you would like to keep in sync
+with the same settings.  You will need the following information from the
+adminisrator of the server:
 
-Strengths:
-.br
-  - Good client support
 .br
-  - Easy setup
+  - The server's URL ("origin", such as "https://tw.example.com")
 .br
-  - Transparent use
-
-Weaknesses:
+  - A client key ("client_key") identifying your tasks
 .br
-  - Tasks are not properly merged
-
+  - An encryption secret ("encryption_secret") used to encrypt and decrypt your tasks. This can be any secret string, and must match for all replicas using the same client key.
```

Comment:
Could we give an example of how to generate this? I know there's probably not a cross-platform way but an example from `/dev/urandom` seems like it would be a good start.

Reply:
My bad, I should have kept reading!